src/tritonserver.cc

```diff
@@ -2755,6 +2755,38 @@ TRITONSERVER_ServerModelStatistics(
     for (const auto& version : mv_pair.second) {
       std::shared_ptr<tc::Model> model;
       RETURN_IF_STATUS_ERROR(lserver->GetModel(mv_pair.first, version, &model));
+
+      // Add memory usage
+      triton::common::TritonJson::Value memory_usage(
```

Comment:
Question: If a backend doesn't support memory usage collection - do we return all 0 - or leave the section out - or indicate via some other value that it is not applicable?